Assets/Packages/com.studio23.ss2.splashscreensystem/Runtime/Data/ThirdPartyData.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+
+namespace Studio23.SS2.SplashScreenSystem.Data
+{
+    [System.Serializable]
+    public class ThirdPartyData : ScriptableObject
+    {
+        public List<ThirdPartyEntry> ThirdPartyEntries = new List<ThirdPartyEntry>();
+    }
+
+    [System.Serializable]
+    public class ThirdPartyEntry
```

Comment:
ThirdPartyData

Reply:
There is already a class named ```ThirdPartyData```, so i rename this to ```ThirdPartyDataEntry```